takwimu/takwimu_ui/src/components/Profile/index.js

```diff
@@ -47,6 +47,12 @@ class Profile extends React.Component {
 
     const title = value === 0 ? 'Data by Topic' : tabs[value].name;
 
+    // Wagtail inserts div/p when RichTextField is empty
+    const hasDescription = () =>
+      description &&
+      description.length > 0 &&
+      description !== '<p></p>' &&
+      description !== '<div class="rich-text"></div>';
```

Comment:
why not just strip all html and then check length to be safe. `.replace(/<[^>]+>/g, '')` should work.

Reply:
[Regular Expresssion, Now You Have Two Problems](https://blog.codinghorror.com/regular-expressions-now-you-have-two-problems/) :sunglasses:

Reply:
I'll have to read this